examples/app_blink.rs

```diff
@@ -0,0 +1,37 @@
+#![feature(no_std, core, start)]
+#![no_std]
+
+extern crate core;
+extern crate zinc;
+
+use zinc::hal::timer::Timer;
+use zinc::hal::lpc17xx::{pin, timer};
+use zinc::hal::pin::GpioDirection;
+use zinc::hal::pin::Gpio;
+use core::option::Option::Some;
+
+#[start]
```

Comment:
Can't get to my build env right now, can you advice why do we need `start` again? Is this being built as a rust binary?